src/inverseRobot.js

```diff
@@ -8,6 +8,27 @@
 
 function inverseRobot(robot) {
   // write code here
+  let tempKyesRobot = {};
+  let tempValueRobot = {};
+
+  tempKyesRobot = Object.keys(robot);
+  tempValueRobot = Object.values(robot);
+
+  for (let i = 0; i < tempValueRobot.length; i++) {
+    if (tempValueRobot.indexOf(tempValueRobot[i]) !== i) {
+      return null;
+    }
+  }
+
+  for (const key in robot) {
```

Comment:
Modifying the input object `robot` is not a good practice as it can lead to unexpected side effects. Instead, create a new object to store the inverted key-value pairs.